scripts/issue/1176/badger_wbtc_bunni_deposit.py

```diff
@@ -0,0 +1,29 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+from pycoingecko import CoinGeckoAPI
+
+
+BADGER_USD = 250_000
```

Comment:
scripts and addrs looks good 

wondering if its better to first run a small tx as a test, check all went alright and then proceed with rest of the funds?

also maybe worth pushing in this pr the piece to enable incentivising in the bunni hh marketplace to have all scripts in place for the treasury decision?

Reply:
yes i like this. can you make `BADGER_USD` a param for the main func, and then we can do something like 25k + 225k or something?

Reply:
> also maybe worth pushing in this pr the piece to enable incentivising in the bunni hh marketplace to have all scripts in place for the treasury decision?

ac40989 - not 100% sure about the way prop is generated for bunni, couldn't find where to verify